src/color-scale/color-scale.js

```diff
@@ -21,6 +21,8 @@ const Self = class ColorScale extends NudeElement {
 			slot: this.shadowRoot.querySelector("slot"),
 			swatches: this.shadowRoot.getElementById("swatches"),
 		};
+
+		this.#hasVs = this.hasAttribute("vs");
```

Comment:
Why are you caching this? Are we reading it that frequently? This could get out of sync if the attribute is added later.